google/gax/api_callable.py

```diff
@@ -88,7 +88,7 @@ def _retryable(a_func, retry):
     total_timeout = (retry.backoff_settings.total_timeout_millis /
                      _MILLIS_PER_SECOND)
 
-    def inner(*args, **kwargs):
```

Comment:
Optional: we should ensure that the "decorator"/transformation functions all look like this, i.e., the final call's `kwargs` comes only from Call{Settings,Options} and not from actual function parameters. This can also be done in a follow-up PR.